### DISCUSSION TITLE
Review my GSOC Proposal - GNOME

### DISCUSSION THREAD

**Discussion by felixfaisal** [EddieHubCommunity/support#1216] [Q&A]
I wanted a review for the GSOC proposal from the community members, I've already submitted so I should be hearing reviews from the mentors sometime soon. 
Also, this could help future or current GSOC aspirants. 
To apply to this org, I wasn't driven by the tech stack, but rather the love I have for Gnome being a Linux user myself. So I learnt whatever possible to apply to this org 

[Proposal](https://drive.google.com/file/d/1FkXV9lz4E1zp0KZ4_CyGmnT_ZnisGPwB/view?usp=sharing)

**Comment by eddiejaoude** [comment]
Sorry for my slow reply @felixfaisal . I had a quick skim through, my initial thought is try to keep the formatting consistent

**Comment by felixfaisal** [comment]
Thank you so much @eddiejaoude for going through it, I did change the formatting before I submitted it. :rocket:

**Reply by eddiejaoude** [reply]
ok great 👍
